### DISCUSSION TITLE
Js for a specific page

### DISCUSSION THREAD

**Discussion by webcreative24** [breezefront/community#72] [1. Help]
Hello. I have added a custom js script in the cms_index_index.xml
`<referenceBlock name="breeze.js">
            <arguments>
                <argument name="bundles" xsi:type="array">
                    <item name="default" xsi:type="array">
                        <item name="items" xsi:type="array">
                            
                            <item name="home-js-file" xsi:type="string">js/index.js</item>
                        </item>
                    </item>
                </argument>
            </arguments>
        </referenceBlock>`
        
        but this script is rendered on all pages. it's possible to add some custom scripts only for specific pages?
        Thanks

**Comment by vovayatsyuk** [comment]
Use cms bundle, or create you own: https://breezefront.com/docs/components#component-registration

The bundle will be rendered if one of its scripts is needed on the page, or if bundle is manually activated.

p.s. You can refer to breeze.js from breeze_default.xml only.
